### ISSUE TITLE
Search across: list multiple exhibits on separate lines

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#1765]
When a result item is included in multiple exhibits, we list those exhibits in a single line, separated by commas. 

I think it would be better to break each exhibit title to a new line. This will make it easier for the user to quickly comprehend that the item belongs to more than one exhibit, and to more easily scan the list for the exhibit they might be most interested in.

(I realize the drawback is if a result item is part of many exhibits, the result item height will increase, but I think it's unlikely we'll often have items in more than a few exhibits, so the tradeoff seems worth it to me.)

<img width="860" alt="Screen Shot 2020-02-26 at 12 27 13 PM" src="https://user-images.githubusercontent.com/101482/75380498-5f918d80-5894-11ea-9a1c-d544fc7507bf.png">

I can't remember offhand a SUL exhibit where we display multi-valued field values on separate lines, but we did do it for the Vatican, where we just used a `<br>` element between values:

<img width="523" alt="Screen Shot 2020-02-26 at 12 34 32 PM" src="https://user-images.githubusercontent.com/101482/75380503-61f3e780-5894-11ea-9601-ad1a6b0a341b.png">

Basically we want a result that is about `0.25rem` between lines.
